Automattic/wp-calypso
# Add gutenboarding import module

#### Changes proposed in this Pull Request
Changes introduce a new Gutenboarding step named "import". The step lives on `/new/import` path. In this stage, idea is to provide UI without business logic. There is a `gutenboarding/import` feature flag which is only turn on for the `development` and `wpcalypso` environments.

For now, available import steps are:
- `List`, query param `?step=list`
- `Capture`, query param `?step=capture`
- `Scanning`, query param `?step=scanning`
- `Ready`, can be reached by adding query param `?step=ready`
- `Ready - No URL`, query param `?step=ready-no-url`
- `Not Ready`, query param `?step=ready-not`

**Desired Import Flow:**
Finish Signup -> Intent Screen -> Click "Import your site content" -> List / Capture / Scanning / Ready screens

#### Testing instructions
Manual testing:
- turn on `gutenboarding/import` feature from [config/development.json](https://github.com/Automattic/wp-calypso/blob/add/gutenboarding-import/config/development.json)
- go to the path `/new/import?step=`
- resize the screen to see how it looks on mobile



Related to #
- [Figma design](https://www.figma.com/file/TAK5FpwTvfbENJDfHc7Aq1/Site-Importing?node-id=1930%3A21706)
